packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -98,19 +98,22 @@ const CodeBlock: React.FC<ICodeBlockProps> = ({
       onChange?.(args.text.split('\n'));
     };
 
-    editorRef.current = servicesRef.current.factoryService.newInlineEditor({
-      host: codeBlockRef.current,
-      model: new CodeEditor.Model({
-        value: defaultValue,
-        mimeType: servicesRef.current.mimeTypeService.getMimeTypeByLanguage({
-          name: language,
-          codemirror_mode: language
+    if (codeBlockRef.current !== null) {
```

Comment:
should we use something like isNotEmpty to check for null and empty string?

Reply:
I don't think so, the following would check if the value is `null`, `undefined`, or an empty value for the type (so for string it would check empty string):
```ts
if (codeBlockRef.current) {
}
```

I just prefer to explicitly check for null/undefined/empty string because I personally find it leads to less bugs